dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java

```diff
@@ -128,7 +128,19 @@
 	 * @throws DotSecurityException The user does not have permissions to perform this action.
 	 */
 	Field save(Field field, User user) throws DotDataException, DotSecurityException;
-	
+
+	/**
+	 * Saves a new Field
+	 *
+	 * @param field
+	 * @param user
+	 * @param reorderIfNeed if it true then reorder all the fields,according to the field to save sort order
+	 * @return
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public Field save(Field field, User user, boolean reorderIfNeed) throws DotDataException, DotSecurityException;
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'save': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6326102)